ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.ts

```diff
@@ -402,6 +401,36 @@ const triggerSet: TriggerSet<Data> = {
       netRegexKo: NetRegexes.startsUsing({ id: '5C7[34]', source: ['헨젤', '그레텔'], capture: false }),
       response: Responses.aoe(),
     },
+    {
+      id: 'Paradigm Hansel/Gretel Bloody Sweep',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '5C5[4567]', source: ['Hansel', 'Gretel'] }),
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
+      durationSeconds: 5,
+      suppressSeconds: 1,
+      alertText: (_data, matches, output) => {
+        if (matches.id === '5C54' || matches.id === '5C55') {
```

Comment:
Would you mind also leaving a comment for a future reader about the two times it can be, so `10` can be explained?

Reply:
Will fix.

Reply:
Thanks, this is great.  It's in the commit message, but having a comment makes this kind of hard-won detail more obvious.  :)